server/controllers/socket.controller.ts

```diff
@@ -175,6 +224,12 @@ export const handleWs = (socket: WebSocket) => {
 				case "Join": {
 					roomCode = data.roomCode;
 					await handleJoin(userId, data);
+					break;
+				}
+				case "StartVoting": {
+					roomCode = data.roomCode;
```

Comment:
Do we need to send roomCode here if we already have it from the Join event?

Reply:
Good point. My inexperience with websockets is showing :-D